Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'solidus', github: 'solidusio/solidus', branch: 'v1.1'
```

Comment:
The thing is, I want to have latest commits from a git branch in this dev branch, not a released gem version from Rubygems. So that's why we can't use the current change.

And, if we update this now, we need to release a 1.1 and 1.2 compatible version of `solidus_i18n` now (and publish branches on GitHub for these releases), so we then can switch the master to 1.3 in master.

Unfortunately I don't have the time right now to do so.

Thanks for your help, it's very much appreciated, but there is a little more to do, then just change versions, sorry.